includes/db-updates.php

```diff
@@ -60,31 +60,37 @@ function wp_stream_update_auto_300( $db_version, $current_version ) {
 	$plugin = wp_stream_get_instance();
 	$plugin->install->install( $current_version );
 
-	$stream_entries = $wpdb->get_results( "SELECT * FROM {$wpdb->base_prefix}stream_tmp" );
+	$starting_row   = 0;
+	$rows_per_round = 5000;
 
-	foreach ( $stream_entries as $entry ) {
-		$context = $wpdb->get_row(
-			$wpdb->prepare( "SELECT * FROM {$wpdb->base_prefix}stream_context_tmp WHERE record_id = %s LIMIT 1", $entry->ID )
-		);
+	while ( ! empty( $stream_entries = $wpdb->get_results( "SELECT * FROM {$wpdb->base_prefix}stream_tmp LIMIT $starting_row, $rows_per_round" ) ) ) {
```

Comment:
@lukecarbis Sure, I just pushed a change that splits that line into two rows. I also had to add another call right before the end of the loop to attempt to retrieve the next set of results. Let me know if this is what you had in mind.